lib/rubocop/cop/style/case_like_if.rb

```diff
@@ -202,6 +214,16 @@ def const_reference?(node)
             name == name.upcase
         end
 
+        def class_reference?(node)
+          return false unless node.const_type?
+
+          name = node.children[1].to_s
```

Comment:
How about using `node.children[1].match? /[[:lower:]]/` instead?

Reply:
The original code checks that const name is not all uppercase, so the code could be written as `!name.match?(/^[[:upper:]]+$/)`, but this does not handle numbers in names and so, the original code looks simpler to me.

Reply:
> checks that const name is not all uppercase

Right. I'm sorry though, I don't see how "not all uppercase" != "has a lowercase", could you provide an example?

Reply:
You're right, sorry. I mistakenly matched that regexp only with the first letter 😄

Reply:
Updated.